packages/core/src/App/Components/Elements/LiveChat/use-livechat.ts

```diff
@@ -101,6 +102,13 @@ const useLiveChat = (has_cookie_account = false) => {
         });
     };
 
+    useEffect(() => {
+        if (isReady && !widget) {
+            setIsReady(false);
+            onHistoryChange();
+        }
+    }, [widget, isReady, onHistoryChange]);
+
```

Comment:
@sergei-deriv let's unite it into one useEffect, the below one has the same dependancies used inside🙂